### PR TITLE
add sub-routes for slack/victorops notification

### PR DESCRIPTION
**Pull request by lauraanddola**
I want to add the sub-routes function (see 2nd routes for filterring check-kyma-installer-* job which will send to slack2 channel) for the slack notification:
routes:
  - receiver: 'null'
    match:
      alertname: Watchdog
  - receiver: "slack"
    continue: true # If set to `false`, it stops after the first matching.
    match_re:
            severity: critical|warning
    routes:
    - match_re:
        job_name: ^(check-kyma-installer-.*|e2e-azure-.*|e2e-gcp-.*)$
      receiver: "slack2"

### CODE REVIEW

File: resources/monitoring/templates/kyma-additions/alertmanager.config.yaml [hunk 1]
```diff
@@ -30,6 +34,9 @@ route:
     continue: true # If set to `false`, it stops after the first matching.
     match_re:
       {{ tpl ( toYaml .Values.global.alertTools.credentials.slack.matchExpression | indent 6) . }}
+    {{- with .Values.global.alertTools.credentials.slack.routes }}
```
Comment (a-thaler) [inline resources/monitoring/templates/kyma-additions/alertmanager.config.yaml]: Why you are not using the same approach as in the match_re by not using the "with" function but only "tpl" without "with"? Any specific reason?

**Comment by lauraanddola** [comment]
thanks to test with helm template *, I found lacking of the `. ` in the code {{ tpl ( toYaml .Values.global.alertTools.credentials.slack.routes) . }}

```
 C02X94NAJGH5:helm_template_0331 i356558$ helm template .  -f values_0401.yaml  --set global.alertTools.credentials.slack.routes='
    - match_re:
        job_name: ^(aa-.*|bb-.*|cc-.*)$
      receiver: "slack2"'
```

**Comment by lauraanddola** [comment]
I recommit the code using {{ .Files.Get "victorOps_subroute.json" }} instead of {{ tpl ( toYaml .Values.global.alertTools.credentials.victorOps.routes) . }}

File: resources/monitoring/templates/kyma-additions/alertmanager.config.yaml (hunk 1 above)
Comment (lauraanddola) [reply resources/monitoring/templates/kyma-additions/alertmanager.config.yaml]: Hello @a-thaler @hisarbalik , finally i try to reuse the {{ with by taking an example from  the kyma/resources/dex/templates/dex-config-map.yaml， and later connector : | could support multiple lines....
```
{{- with .Values.connectors }}
    connectors:
    {{- tpl . $ | nindent 4 }}
  {{- end }}
```
could u help to take a review? thanks.

File: docs/monitoring/05-01-alertmanager.md [hunk 2]
```diff
@@ -18,8 +18,9 @@ This table lists the configurable parameters, their descriptions, and default va
 | **global.alertTools.credentials.slack.apiurl** | Specifies the URL endpoint which sends alerts triggered by Prometheus rules.  | None |
 | **global.alertTools.credentials.slack.channel** | Refers to the Slack channel which receives notifications on new alerts. | None |
 | **global.alertTools.credentials.slack.matchExpression** | Notifications will be sent only for those alerts whose labels match the specified expression.  | "severity: critical" |
+| **global.alertTools.credentials.slack.routes** | Refers to the Slack subroutes which receives notifications on new filtered alerts. | None |
 | **global.alertTools.credentials.victorOps.routingkey** | Defines the team routing key in [VictorOps](https://help.victorops.com/). | None |
 | **global.alertTools.credentials.victorOps.apikey** | Defines the team API key in VictorOps. | None |
 | **global.alertTools.credentials.victorOps.matchExpression** | Notifications will be sent only for those alerts whose labels match the specified expression.  | "severity: critical" |
-
+| **global.alertTools.credentials.victorOps.routes** | Refers to the victorOps subroutes which receives notifications on new filtered alerts. | None |
```
Comment (bszwarc) [inline docs/monitoring/05-01-alertmanager.md:25]:
```suggestion
| **global.alertTools.credentials.victorOps.routes** | Defines the victorOps subroutes which receive notifications on new filtered alerts. | None |
```

**Pull request merged**